images/installer/Dockerfile.upi.ci

```diff
@@ -88,6 +88,7 @@ RUN mkdir /output && HOME=/output && \
     ibmcloud plugin install cloud-internet-services -f && \
     cp /usr/local/bin/ibmcloud /bin/ibmcloud && \
     rm -f $HOME/.curlrc && \
+    chmod -R g=u "$HOME/.bluemix/" && \
```

Comment:
Per the results, we need revert the line `RUN chown -R 1000:1000 /output` to `RUN chown 1000:1000 /output`

Reply:
checked with the local test

> podman build -f Dockerfile.upi.ci --authfile ./pull-secret 
> podman run --rm -it <image id>

_chown -R 1000:1000 /output_  is OK